### ISSUE TITLE
[feature request/idea] swap tiled windows by dragging

### ISSUE THREAD

**Issue by clapbr** [esjeon/krohnkite#40]
Just an idea, close if you think it's a bad one since the argument is based on what I feel. 

This is a really useful feature I saw on other tiling implementations.

When I drag windows around my muscle memory expects the windows to swap places instead of setting it as float, which is an action I almost never have to do this way since you can pre-configure floating applications or simply use the hotkey.

It's a really intuitive way to quickly organize the layout, mainly in situations where you already are using the mouse a lot.

**Comment by gbytedev** [comment]
Maybe the the [kwin-tiling](https://github.com/kwin-scripts/kwin-tiling) script can be used as reference for this specific feature.

**Comment by esjeon** [comment]
I've just implemented it with 73d22addcae96cd42fa9c79fc467f35c913d83ba. Krohnkite now swaps tiles just like kwin-tiling does: aim with your window. lol. I actually tried many other methods, but this one was the sanest one, since KWin doesn't expose mouse position. **BUT....**

I also implemented a hack that polls actual mouse position using `xdotool`. This has to be enabled in the config dialog, under `Options` tab. KWin will hiccup sometimes, but works just fine.
